tests/functional/companies/delete.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { Roles } from '#apps/shared/contracts/roles'
+import Professional from '#apps/shared/models/professional'
+import { test } from '@japa/runner'
+
+test.group('Companies delete', () => {
+  test('should sucessfully delete a company with valid id', async ({ client }) => {
+    const professional = await Professional.firstOrFail()
+
+    const response = await client
+      .delete('/v1/companies/9890150790490362881976491')
```

Comment:
When you would test, you should to make factory instead seeder.
Then call your factory like `const model = await Factory.make(…)` to create model. instance.

So you can get `model.id` to replace `9890150790490362881976491`

Reply:
For the moment I've made a simple seeder, maybe in time we'll do something cleaner with Factory.